src/Annotations/Command/QueueWatchCommand.php

```diff
@@ -43,7 +44,9 @@ protected function process(InputInterface $input, QueueItem $item, $entity = nul
     {
         if ($entity instanceof Profile) {
             $id = $entity->getIdentifier()->getId();
-            $emails = $entity->getEmailAddresses();
+            $emails = $entity->getEmailAddresses()->map(function (AccessControl $accessControl) {
```

Comment:
I'd not rely on the client having only requested public data, and add a `filter()`. Avoids future mistakes.